src/auth-service/controllers/join.js

```diff
@@ -91,7 +91,7 @@ const join = {
         let nestedErrors = validationResult(req).errors[0].nestedErrors;
         return badRequest(
           res,
-          "bad request errors",
+          "User does not exist",
```

Comment:
Thanks @BenjaminSsempala for this.

I am not sure about this change.....
Is this "function" for many other use cases that are not just about users?

**Context:**
Auth service handles more than just users, it does handle:
1. Inquiries
2. candidates
3. organisations/networks and
4. defaults

Please help shed more light on this?

Reply:
Hello @Baalmart 
I made this change inside the `lookupfirebase` function which I assumed is only utilized to look up users in firebase for the mobile app, although I could be wrong. With that assumption in mind, I thought to make the message more specific since it would only be for users. Also, some additional context may be acquired from the information in this [jira card](https://airqoteam.atlassian.net/browse/MOB-146).

Reply:
Thanks @BenjaminSsempala for the clarification, much appreciated!